src/core/business-logic/core-units.ts

```diff
@@ -240,6 +240,50 @@ export const getLast3ExpenditureValuesFromCoreUnit = (cu: CoreUnitDto) => {
   return result;
 };
 
+export const getCurrentOrLastMonthWithData = (budgetStatements: BudgetStatementDto[]) => {
+  const currentMonth = DateTime.now().set({
```

Comment:
You can use `.startOf('month')`, it will reset everything to 0 after the month field (no need to change)

Reply:
Agreed